### DISCUSSION TITLE
No scheduler?

### DISCUSSION THREAD

**Discussion by jaspov** [tnmendes/watt#89] [General]
very nice app however can’t find a scheduler like on the tp-link
Kasa app. Am I missing something? Have my coffee machine scheduled each morning.

**Comment by tnmendes** [comment]
Hey @jaspov,
Thanks for liking the Watt app, however the Watt app doesn't have a scheduler implemented.

**Comment by jaspov** [comment]
Thanks for the quick response. Is this a feature that might get added? Prefer to have one app per device and yours is clearly better. On 5 Nov 2022, at 11:52, Tiago Mendes ***@***.***> wrote:﻿
Hey @jaspov,
Thanks for liking the Watt app, however the Watt app doesn't have a scheduler implemented.

**Comment by tnmendes** [comment]
Sorry at the moment there are no plans to add this feature but you can open the Kasa app and configure it once and after it's configured, you don't need to open Kasa again ‎😃‎😃‎😃

**Comment by jaspov** [comment]
Thanks however sometimes need to change it when I wake up early, on holidays and so on. On 5 Nov 2022, at 15:56, Tiago Mendes ***@***.***> wrote:﻿
Sorry at the moment there are no plans to add this feature but you can open the Kasa app and configure it once and after it's configured, you don't need to open Kasa again ‎😃‎😃‎😃

**Comment by aspwlf** [comment]
I wish there was a timer bruh. One of the only features I used too. The only reason I use this app is literally becahse The TP LINK app is fuggin 500MB!!!!!!! This app is like 45mb 
That’s the only reason I use it LOL
